src/main/java/be/seeseemelk/mockbukkit/inventory/InventoryMock.java

```diff
@@ -1,39 +1,37 @@
 package be.seeseemelk.mockbukkit.inventory;
 
-import static org.junit.jupiter.api.Assertions.assertTrue;
-import static org.junit.jupiter.api.Assertions.fail;
-
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.List;
-import java.util.ListIterator;
-import java.util.Objects;
+import java.util.*;
 import java.util.function.Predicate;
 import java.util.stream.Collectors;
 
+import be.seeseemelk.mockbukkit.MockBukkit;
+import com.google.common.collect.Lists;
 import org.apache.commons.lang.Validate;
+import org.bukkit.Bukkit;
 import org.bukkit.Location;
 import org.bukkit.Material;
 import org.bukkit.entity.HumanEntity;
+import org.bukkit.entity.Player;
 import org.bukkit.event.inventory.InventoryType;
 import org.bukkit.inventory.Inventory;
 import org.bukkit.inventory.InventoryHolder;
+import org.bukkit.inventory.InventoryView;
 import org.bukkit.inventory.ItemStack;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
 import be.seeseemelk.mockbukkit.UnimplementedOperationException;
 
-public class InventoryMock implements Inventory
-{
+import static org.junit.jupiter.api.Assertions.*;
+
+public class InventoryMock implements Inventory {
```

Comment:
You broke the formatting here.

```suggestion
public class InventoryMock implements Inventory
{
```